README.md

```diff
@@ -25,6 +25,8 @@ Note this package is in preview and it will not work with your Tecton installati
 
 ## Changelog
 
+- 0.1.0 Add 2 new operators to support triggering Feature Table ingestion jobs 
```

Comment:
s/Add/Adds

Reply:
**Add** matches **Support** in v0.0.3

Reply:
I think it should probably be `added`. I think past tense is correct, but I don't think incorrect tense in past patch notes should stop us